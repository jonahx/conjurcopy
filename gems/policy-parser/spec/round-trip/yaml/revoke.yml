- !revoke
  role: !group developers
  member: !user alice

