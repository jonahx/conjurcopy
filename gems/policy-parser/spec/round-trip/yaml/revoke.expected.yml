---
- !revoke
  member: !user
    id: alice
  role: !group
    id: developers
